parcels/particleset.py

```diff
@@ -122,6 +105,46 @@ def cptr(i):
         else:
             raise ValueError("Latitude and longitude required for generating ParticleSet")
 
+    @classmethod
+    def from_list(cls, grid, pclass, lon, lat):
+        """Initialise the ParticleSet from start/finish coordinates
+        with equidistant spacing
+
+        :param grid: :mod:`parcels.grid.Grid` object from which to sample velocity
+        :param pclass: mod:`parcels.particle.JITParticle` or :mod:`parcels.particle.ScipyParticle`
+                 object that defines custom particle
+        """
+        return cls(grid=grid, pclass=pclass, lon=lon, lat=lat)
+
+    @classmethod
+    def from_line(cls, grid, pclass, start, finish, size):
+        """Initialise the ParticleSet from start/finish coordinates
+        with equidistant spacing
+
+        :param grid: :mod:`parcels.grid.Grid` object from which to sample velocity
+        :param pclass: mod:`parcels.particle.JITParticle` or :mod:`parcels.particle.ScipyParticle`
+                 object that defines custom particle
+        :param start: Starting point for initialisation of particles on a straight line.
+        :param finish: End point for initialisation of particles on a straight line.
+        :param size: Initial size of particle set
+        """
+        lon = np.linspace(start[0], finish[0], size, dtype=np.float32)
+        lat = np.linspace(start[1], finish[1], size, dtype=np.float32)
```

Comment:
Depending on the mesh type of `grid`, should we make sure that particles are genuinely evenly spaced if `start` and `finish` span long distances?